cfme/tests/services/test_provision_stack.py

```diff
@@ -164,12 +163,6 @@ def template(provider, provisioning, dialog_name):
 
     yield template
 
-    try:
```

Comment:
I'm not sure what you mean by your comment. Its scoped for function, so for each test it will create+delete the template that's used?